### PR TITLE
Better install modal UI

### PR DESCRIPTION
**Pull request by CPatchane**
Improve space usage in the modal.

### CODE REVIEW

**Comment by CPatchane** [comment]
[(1) [Store] Revoir les styles de la modale d'inter apps des permissions](https://trello.com/c/xPZPorYq/1060-1-store-revoir-les-styles-de-la-modale-dinter-apps-des-permissions)

File: src/styles/install.css [hunk 1]
```diff
@@ -17,8 +16,18 @@
   justify-content: center;
 }
 
-.sto-install .sto-install-header {
-  padding-top: 2rem;
+.sto-install-content {
+  overflow-y: scroll;
```
Comment (gregorylegarec) [inline src/styles/install.css]: Should be `auto` here, otherwise the scrollbar will always be visible.

**Comment by gregorylegarec** [comment]
LGTM except one little change.

**Comment by gregorylegarec** [comment]
Also, there is an empty space on the right

capture d ecran 2018-06-19 a 09 44 15

**Comment by CPatchane** [comment]
@gregorylegarec I don't have this issue in the store, I think this is due to the Intent modal here :/

**Comment by gregorylegarec** [comment]
Thanks @CPatchane 👌

**Pull request merged**